### PR TITLE
Feature/custom rollup config

### PR DESCRIPTION
**Pull request by sandeep-vedam**
Provided support for custom rollup config file

### CODE REVIEW

**Review by wouterlucas — APPROVED** [review]
nice! 🕺

File: src/helpers/build.js [hunk 1]
```diff
@@ -217,15 +217,25 @@ const bundleAppRollup = (folder, metadata, type, options) => {
...
     '--name',
     makeSafeAppId(metadata),
-    '--preserveSymlinks',
   ]
 
+  const rollupConfig = require(path.join(__dirname, `../configs/rollup.${type}.config.js`))
+  // Check if 'input' property is not present in the rollupConfig object
+  if (!('input' in rollupConfig)) {
+    //if 'input' is not present, push the input option and location of source file to the args
+    args.push(`--input , ${path.join(process.cwd(), enterFile)}`)
```
[the remark this answers is not in the document]
Comment (ricardo-a-alves-alb) [reply src/helpers/build.js:230]: It can be further simplified:

```js
args.push('--input', path.join(process.cwd(), enterFile))
```

**Review by iamtabrezkhan — APPROVED** [review]

**Comment by iamtabrezkhan** [comment]
Hey guys, when can we release this change?

File: src/configs/rollup.es5.config.js [hunk 2]
```diff
@@ -39,8 +39,15 @@ const minify = require('rollup-plugin-terser').terser
 const license = require('rollup-plugin-license')
 const os = require('os')
 const extensions = ['.js', '.ts', '.mjs']
+const deepMerge = require('deepmerge')
 
-module.exports = {
+let customConfig
+
+if (process.env.LNG_CUSTOM_ROLLUP === 'true') {
+  customConfig = require(path.join(process.cwd(), 'rollup.es5.config'))
```
Comment (ricardo-a-alves-alb) [inline src/configs/rollup.es5.config.js:47]: What about the _env_?

File: src/configs/rollup.es6.config.js [hunk 3]
```diff
@@ -37,8 +37,15 @@ const minify = require('rollup-plugin-terser').terser
 const license = require('rollup-plugin-license')
 const os = require('os')
 const extensions = ['.js', '.ts']
+const deepMerge = require('deepmerge')
 
-module.exports = {
+let customConfig
+
+if (process.env.LNG_CUSTOM_ROLLUP === 'true') {
+  customConfig = require(path.join(process.cwd(), 'rollup.es6.config'))
```
Comment (ricardo-a-alves-alb) [inline src/configs/rollup.es6.config.js:45]: What about the _env_?

**Review by ricardo-a-alves-alb — CHANGES_REQUESTED** [review]

File: src/configs/rollup.es5.config.js (hunk 2 above)
Comment (sandeep-vedam) [reply src/configs/rollup.es5.config.js:47]: Hey @ricardo-a-alves-alb ,
The ones that are mentioned in the env will be ignored here and the custom config takes the priority here.

File: src/configs/rollup.es6.config.js (hunk 3 above)
Comment (sandeep-vedam) [reply src/configs/rollup.es6.config.js:45]: The ones that are mentioned in the env will be ignored here and the custom config takes the priority here.

**Review by michielvandergeest — APPROVED** [review]
LGTM

**Pull request merged**